meetings/2018-06-13.md

```diff
@@ -0,0 +1,110 @@
+# Node.js Foundation Technical Steering Committee (TSC) Meeting 2018-06-13
+
+## Links
+
+* **Recording**:  https://www.youtube.com/watch?v=OxjWOgC1U9k
+* **GitHub Issue**: https://github.com/nodejs/TSC/issues/551
+
+## Present
+
+* Anna Henningsen @addaleax (TSC)
+* Сковорода Никита Андреевич @ChALkeR (TSC)
+* Franziska Hinkelmann @fhinkel (TSC)
+* Jeremiah Senkpiel @Fishrock123 (TSC)
+* Michael Dawson @mhdawson (TSC)
+* Myles Borins @MylesBorins (TSC)
+* Rod Vagg @rvagg (TSC)
+
+## Agenda
+
+### Announcements
+ 
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+### nodejs/node
+
+* process: add allowedEnvironmentNodeFlags property [#19335](https://github.com/nodejs/node/pull/19335)
+  * Had some initial discussions, will pick up again next time when we have more
+    people and more TSC members have reviewed.
+  * seems likes needs TSC champion to push forward.
+
+* Nominating bcoe as a collaborator [#20242](https://github.com/nodejs/node/issues/20242)
+  * Seems like everybody on the issue is +1, any objections.  Myles will find somebody to
+    Onboard.
+
+### nodejs/TSC
+
+* Tracking issue for updating TSC on Board Meetings [#476](https://github.com/nodejs/TSC/issues/476)
+  * Nothing to add since last board meeting
+  * Work on getting CommComm charter in place for Director etc.
+
+* Strategic Initiatives - Tracking Issue [#423](https://github.com/nodejs/TSC/issues/423)
+  * Modules
+    * Have gotten through a larger number of the use cases
+    * PR open for meta require
+    * Starting to talk about deadlines
+    * Main contentious topics is about transparent interop, module specifier resolution
+    * Package name maps proposal from Domenic, to allow install time tooling to generate
+      manifest to be consume by browser to allow bare imports.  May provide path to being
+      to use modules in both browsers and Node.js without changes. Does not affect CJS
+
+  * N-API
+     * Continued push on doc for node-addon-api
+     * Looks like we’ll have workshop and a few talks at Node Summit
+     * Proposal for adding new APIs as experimental
+     * Proposal for guidance on adding new APIs
+
+  * OpenSSL Evolution
+    * Pull request 21282, would some some additional opinion.  Security issue with CVE
```

Comment:
Not sure what that first sentence is supposed to be (`would some some additional opinion` should be....something else, but what?)